website/docs/r/api_management_property.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_property"
+sidebar_current: "docs-azurerm-resource-api-management-property-x"
+description: |-
+  Manages an API Management Property.
+---
+
+# azurerm_api_management_property
+
+Manages an API Management Property.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West US"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  publisher_name      = "pub1"
+  publisher_email     = "pub1@email.com"
+
+  sku {
+    name     = "Developer"
+    capacity = 1
+  }
+}
+
+resource "azurerm_api_management_property" "example" {
+  name                = "example-apimg"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  api_management_name = "${azurerm_api_management.example.name}"
+  display_name        = "ExampleProperty"
+  value               = "Example Value"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the API Management Property. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which the API Management Property should exist. Changing this forces a new resource to be created.
+
+* `api_management_name` - (Required) The name of the [API Management Service](api_management.html) in which the API Management Property should exist. Changing this forces a new resource to be created.
+
+* `display_name` - (Required) The display name of this API Management Property.
+
+* `value` - (Required) The value of this API Management Property.
+
+* `secret` - (Optional) Specifies whether the API Management Property is secret. Valid values are `true` or `false`. The default value is `false`.
```

Comment:
I feel like it could be worth a yellow info box here explaining this isn't making it sensitive within Terraform e.g.

```
~> **NOTE:** setting the field `secret` to `true` doesn't make this field sensitive in Terraform, instead it ...
```

Reply:
Done!